lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -59,7 +62,32 @@ export class RefreshTokenClient extends BaseClient {
             undefined,
             true,
             request.forceCache
-        );
+        ).then((result: AuthenticationResult) => {
+            if (response.body?.refresh_token) {
+                atsMeasurement?.endMeasurement({
+                    success: true,
+                    refreshTokenSize: response?.body?.refresh_token.length || 0
+                });
+            }
+            else{
+                // no refresh token is returned
+                atsMeasurement?.endMeasurement({
+                    success: true,
+                    refreshTokenSize: undefined
+                });
+            }
```

Comment:
Something like this might be more concise
```suggestion
            let refreshTokenSize;
            if (response.body?.refresh_token) {
                refreshTokenSize = response.body.refresh_token.length || 0;
            }
            atsMeasurement?.endMeasurement({
                success: true,
                refreshTokenSize: refreshTokenSize
            });
```

Reply:
Thanks. This definitely is cleaner, would update. @tnorling